### PR TITLE
Add param "isconstant" to `stumpy/mmotifs.py`

### PR DESCRIPTION
**Pull request by NimaSarajpoor**

### CODE REVIEW

File: tests/test_mmotifs.py [hunk 1]
```diff
@@ -193,3 +193,37 @@ def test_mmotifs_two_motif_pairs_max_motifs_2(T):
...
+    motif_subspaces_ref = [np.array([1])]
+    motif_mdls_ref = [np.array([232.0, 250.57542476, 260.0, 271.3509059])]
+
+    m = 4
+    excl_zone = int(np.ceil(m / config.STUMPY_EXCL_ZONE_DENOM))
+
+    # The following `T_subseq_isconstant` is basically equivalent to
+    # `T_subseq_isconstant=None` (default). The goal is to test its
+    # functionality.
+    T_subseq_isconstant = [
```
Comment (NimaSarajpoor) [inline tests/test_mmotifs.py:211]: Due to the complexity of test functions in `mmotifs`, I created this [naive-like] test function to test the param `T_subseq_isconstant`. Note that `T_subseq_isconstant` is provided as a list consisting of items that reflect the "default" behavior.

File: tests/test_mmotifs.py [hunk 2]
```diff
@@ -193,3 +193,37 @@ def test_mmotifs_two_motif_pairs_max_motifs_2(T):
...
+        None,
+        naive.rolling_isconstant(T[1], m),
+        None,
+        naive.is_ptp_zero_1d,
+    ]
+
+    P, I = naive.mstump(T, m, excl_zone)
```
Comment (NimaSarajpoor) [inline tests/test_mmotifs.py:218]: After merging the PR #871 , I will pass `T_subseq_isconstant` to `naive.mstump`.

**Review by NimaSarajpoor — COMMENTED** [review]
@seanlaw 
Not sure if the new test function suffices or not. Other than that, we are almost good in this PR.

**Comment by seanlaw** [comment]
Looks good to me. Please merge #871 and let's see if anything changes

**Comment by NimaSarajpoor** [comment]
I think this PR is ready. Plz let me know if there is any thing that I should address.

**Comment by seanlaw** [comment]
@NimaSarajpoor Thank you for this PR!

**Pull request merged**